src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -5564,10 +5567,10 @@ void func_808335B0(PlayState* play, Player* this) {
 }
 
 s8 D_8085D090[] = {
-    PLAYER_MWA_STAB_1H,
-    PLAYER_MWA_RIGHT_SLASH_1H,
-    PLAYER_MWA_RIGHT_SLASH_1H,
-    PLAYER_MWA_LEFT_SLASH_1H,
+    PLAYER_MWA_STAB_1H,        // PLAYER_STICK_DIR_FORWARD
+    PLAYER_MWA_RIGHT_SLASH_1H, // PLAYER_STICK_DIR_LEFT
+    PLAYER_MWA_RIGHT_SLASH_1H, // PLAYER_STICK_DIR_BACKWARD
+    PLAYER_MWA_LEFT_SLASH_1H,  // PLAYER_STICK_DIR_RIGHT
```

Comment:
Worth noting that the left/right directions don't line up here, so maybe it's worth taking another look at MWA soon

Reply:
I could be mistaken, but
```c
Player_PlaySfx(this, ((controlStickDirection << 0xE) == (PLAYER_STICK_DIR_BACKWARD << 0xE)) ? NA_SE_PL_ROLL : NA_SE_PL_SKIP);
```
also seems a little strange right?

Reply:
Why not add a few TODO comments noting this inconsistency?

Reply:
I'm not sure it's that inconsistent. This function deals with either a side hop (which plays `NA_SE_PL_SKIP`) or a backflip (which reuses the sfx `NA_SE_PL_ROLL`). `NA_SE_PL_ROLL` is also used for a regular roll forward, so it's just a multi-use sfx

Reply:
That makes sense. Thanks for the clarification. Should probably add a TODO to verify the MWA though.

Reply:
done!